Frameworks/UIKit.Xaml/ProgressView.xaml

```diff
@@ -0,0 +1,19 @@
+﻿<Grid
+    x:Class="UIKit.Xaml.ProgressView"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:local="using:UIKit.Xaml"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    mc:Ignorable="d"
+    x:Name="Root">
+    <ProgressBar Name="progressBar"
+                 IsIndeterminate="False"
+                 Minimum="0.0"
+                 Maximum="1.0"
+                 Height="{x:Bind Height, Mode=OneWay}"
+                 Width="{x:Bind Width, Mode=OneWay}"
```

Comment:
The height and width are bound to the size of the containing grid, so that the ProgressBar size will always be updated with the frame is resized on UIProgressView.
Removed IsIndeterminate, since that's the default value.
Minimum and Maximum are supposed to default to 0 and 1 according to the docs, but I was actually seeing maximum default to 100. So this is needed to ensure the min/max matches iOS.

---
In reply to: [114915819](https://github.com/Microsoft/WinObjC/pull/2668#discussion_r114915819) [](ancestors = 114915819)